src/app/core/services/repo-creator.service.ts

```diff
@@ -18,9 +20,43 @@ export const BUG_REPORTING_INVALID_ROLE =
 export class RepoCreatorService {
   constructor(
     private githubService: GithubService,
-    private userService: UserService
+    private userService: UserService,
+    private phaseFixConfirmationDialog: MatDialog
```

Comment:
```suggestion
    private repoCreationConfirmationDialog: MatDialog
```

I think we can use this chance to adopt a name with a more obvious meaning here.
("phase fix" seems ambiguous)

Reply:
:+1: